frontend/src/components/ManualConfig.vue

```diff
@@ -0,0 +1,66 @@
+ <template>
```

Comment:
Please double check indentation spacing here - looks like it has one extra space.

Reply:
Formatted.